wsf_scraping/pipelines.py

```diff
@@ -74,6 +58,7 @@ def check_keywords(self, item, spider_name, base_pdf_path):
 
             pdf_file, pdf_text = parse_pdf_document(f)
 
+            # If the PDF couldn't be converted, still remove the pdf file
             if not pdf_file:
                 os.remove(item['pdf'])
```

Comment:
`os.remove()` is a rare, rare synonym in Python for the more commonly used (in UNIX, at least) `unlink()`. No need to change, just an FYI.

Reply:
>the unlink name is its traditional Unix name

I changed it 😄